components/server/src/gitlab/api.ts

```diff
@@ -128,16 +115,16 @@ export namespace GitLab {
             return !!something && something.name === "GitLabApiError";
         }
         export function isNotFound(error: ApiError): boolean {
-            return !!error.httpError?.description.startsWith("404");
```

Comment:
I've seen the `startsWith` of undefined before, and it was related to GitLab contexts. 🙏🏻 hopefully that's the cause